lib/ui/screen/login_screen.dart

```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
-                        onEditingComplete: () {
-                          _accountFocus.unfocus();
-                          FocusScope.of(context).requestFocus(_passwordFocus);
-                        },
-                        validator: (value) => _validatorAccount(value),
-                        decoration: InputDecoration(
-                          hintText: R.current.account,
-                          errorStyle: const TextStyle(
-                            height: 0,
-                            fontSize: 0,
+                child: AutofillGroup(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Material(
+                        elevation: 2,
+                        borderRadius: const BorderRadius.all(Radius.circular(32)),
+                        child: TextFormField(
+                          controller: _accountControl,
+                          cursorColor: Colors.blue[800],
+                          textInputAction: TextInputAction.next,
+                          focusNode: _accountFocus,
+                          autofillHints: const [AutofillHints.username],
+                          onEditingComplete: () {
+                            _accountFocus.unfocus();
+                            FocusScope.of(context).requestFocus(_passwordFocus);
+                          },
+                          validator: (value) => _validatorAccount(value),
+                          decoration: InputDecoration(
+                            hintText: R.current.account,
+                            errorStyle: const TextStyle(
+                              height: 0,
+                              fontSize: 0,
+                            ),
+                            prefixIcon: const Icon(
+                              Icons.account_circle,
+                              color: Colors.grey,
+                            ),
+                            border: InputBorder.none,
                           ),
-                          prefixIcon: const Icon(
-                            Icons.account_circle,
-                            color: Colors.grey,
-                          ),
-                          border: InputBorder.none,
                         ),
                       ),
-                    ),
-                    const SizedBox(
-                      height: 4,
-                    ),
-                    if (_accountErrorMessage.isNotEmpty)
-                      Padding(
-                        padding: const EdgeInsets.only(left: 16),
-                        child: Text(
-                          _accountErrorMessage,
-                          style: const TextStyle(
-                            fontSize: 12,
-                            color: Colors.red,
+                      const SizedBox(
+                        height: 4,
+                      ),
```

Comment:
[nit]

Could you help us remove the trailing comma for widgets with single properties?
Although we don't have very strict conventions, it makes code look nicer in my opinion.

```suggestion
                      const SizedBox(height: 4),
```

Reply:
Done in [9d92f57](https://github.com/NEO-TAT/tat_flutter/pull/185/commits/9d92f57c53a7bda5eebc22af86d0d46c1833ba45).